src/WWT.Providers/QueryRequestProvider.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace WWT.Providers
+{
+    internal static class WwtContextExtensions
```

Comment:
The name of this file seems out of sync with the class name.

Reply:
True! I've removed it to simplify the refactoring